docs/notebooks/unfinished/example_measure_elzerman_readout.py

```diff
@@ -0,0 +1,54 @@
+from qtt.measurements.acquisition import UHFLIScopeReader, UHFLIStimulus
+from matplotlib import pyplot as plt
+
+
+# INITIALIZE THE SCOPE READER
+
+device_id = 'dev2338'
+time_period = 2
+trigger_enabled = True
+input_channel = 1
+input_channel_attribute = 'Demod 1 R'
+scope_reader = UHFLIScopeReader(address=device_id)
+sampling_rate = scope_reader.get_nearest_sample_rate(27e3)
```

Comment:
The `sampling_rate` is not used in the script. Should it be set on the `UHFLIScopeReader`?